src/main/java/DatabaseInteractin/DataBaseInteracting.java

```diff
@@ -0,0 +1,75 @@
+package DatabaseInteractin;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataBaseInteracting {
+  private final Connection connection = DataBaseConnection.getConnection();
+
+  public DataBaseInteracting() throws SQLException {
+  }
+
+  public PreparedStatement createPrepareStatement() throws SQLException {
+    String sql = "SELECT * FROM products";
+    return connection.prepareStatement(sql);
+  }
+
+  public void addProductToDatabase(String n, int s, float p) throws SQLException {
+    try {
+      String sql = "INSERT INTO products (name, stock, price) VALUES (?,?,?)";
+      PreparedStatement preparedStatement = connection.prepareStatement(sql);
+      preparedStatement.setString(1, n);
+      preparedStatement.setInt(2, s);
+      preparedStatement.setFloat(3, p);
+      preparedStatement.executeUpdate();
+    } catch (SQLException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public ResultSet getProductsById(int id) throws SQLException {
+    ResultSet resultSet = null;
+    try {
+      String sql = "select * from products where id =" + id;
```

Comment:
Keep consistent casing for your SQL queries too please, on line 23 you are using uppercase, here it's lowercase. (uppercase is prefered)